src/ril_binder_radio.c

```diff
@@ -283,6 +309,58 @@ ril_binder_radio_apn_types_for_profile(
     }
 }
 
+static
+const char *
+radio_pdp_protocol_type_to_str(enum radio_pdp_protocol_type type)
+{
+    switch (type) {
+    case RADIO_PDP_PROTOCOL_IP:
+        return RIL_PROTO_IP_STR;
+    case RADIO_PDP_PROTOCOL_IPV6:
+        return RIL_PROTO_IPV6_STR;
+    case RADIO_PDP_PROTOCOL_IPV4V6:
+        return RIL_PROTO_IPV4V6_STR;
+    default:
+        return "";
+    }
+}
+
+static
+gboolean
+ril_binder_radio_interface_supported(
+    RADIO_INTERFACE interface)
+{
+    switch (interface) {
+    case RADIO_INTERFACE_1_0:
+    case RADIO_INTERFACE_1_2:
+    case RADIO_INTERFACE_1_4:
+        return TRUE;
+    default:
+        return FALSE;
+    }
+}
+
+static
+const char*
+ril_binder_radio_interface_name(
+    RADIO_INTERFACE interface)
+{
+    switch (interface) {
+    case RADIO_INTERFACE_1_0:
+        return "radio@1.0";
+    case RADIO_INTERFACE_1_1:
+        return "radio@1.1";
+    case RADIO_INTERFACE_1_2:
+        return "radio@1.2";
+    case RADIO_INTERFACE_1_3:
+        return "radio@1.3";
+    case RADIO_INTERFACE_1_4:
+        return "radio@1.4";
+    default:
+        return NULL;
+    }
```

Comment:
It's a good idea to now have a default. Doing this instead:
```c
static
const char*
ril_binder_radio_interface_name(
    RADIO_INTERFACE interface)
{
    switch (interface) {
    case RADIO_INTERFACE_1_0: return "radio@1.0";
    case RADIO_INTERFACE_1_1: return "radio@1.1";
    case RADIO_INTERFACE_1_2: return "radio@1.2";
    case RADIO_INTERFACE_1_3: return "radio@1.3";
    case RADIO_INTERFACE_1_4: return "radio@1.4";
    case RADIO_INTERFACE_COUNT:
        break;
    }
    return NULL;
}
```
Would produce "enumeration value not handled in switch" warning when a new enumeration value is introduced.